### ISSUE TITLE
"Nightly" Libretro Buildbot cores for armhf and armv7-neon-hf are 2.5 years old

### ISSUE THREAD

**Issue by CFTBL** [libretro/RetroArch#15154]
It appears that the "nightly" builds at https://buildbot.libretro.com/nightly/linux/armv7-neon-hf/latest/ and https://buildbot.libretro.com/nightly/linux/armhf/latest/ aren't getting built (at least according to their  timestamps). If those timestamps are accurate, it means anytime you use RetroArch's online updater to update cores on a Raspberry Pi 4, you're getting cores that are 2.5 years old.

**Comment by hizzlekizzle** [comment]
Yep, that's true. There is a github repo that hosts up-to-date builds for those arches here: https://github.com/christianhaitian/retroarch-cores

but we can't speak for the integrity of them, so it would be irresponsible to just copy/paste them into the online updater for distribution, unfortunately.

**Comment by zoltanvb** [comment]
I have also compiled some cores that are available for download:
https://zoltanvb.github.io/armv7-hf-neon/

**Comment by Ninoh-FOX** [comment]
> I have also compiled some cores that are available for download:
> https://zoltanvb.github.io/armv7-hf-neon/

Not work, I retroarch for miyoo is empy.

**Comment by MeisterLLD** [comment]
> I have also compiled some cores that are available for download: https://zoltanvb.github.io/armv7-hf-neon/

I've tried your flycast_libretro, it does not work right now (goes back to EmulationStation on Retropie)

**Comment by zoltanvb** [comment]
Thanks, added a separate issue to follow that one.

**Comment by zoltanvb** [comment]
> > I have also compiled some cores that are available for download:
> > https://zoltanvb.github.io/armv7-hf-neon/
> 
> Not work, In retroarch for miyoo is empy.

Which model? Actually for miyoo, there are cores on the buildbot, such as: http://buildbot.libretro.com/nightly/dingux/miyoo-arm32/latest/

**Comment by Ninoh-FOX** [comment]
> > > I have also compiled some cores that are available for download:
> > > https://zoltanvb.github.io/armv7-hf-neon/
> > 
> > Not work, In retroarch for miyoo is empy.
> 
> Which model? Actually for miyoo, there are cores on the buildbot, such as: http://buildbot.libretro.com/nightly/dingux/miyoo-arm32/latest/

But this is for old miyoo models, the last model (mini and mini plus) work with armhf.

**Comment by slaminger** [comment]
> Yep, that's true. There is a github repo that hosts up-to-date builds for those arches here: https://github.com/christianhaitian/retroarch-cores
> 
> but we can't speak for the integrity of them, so it would be irresponsible to just copy/paste them into the online updater for distribution, unfortunately.

can't speak to the integrity of them in what way?

**Comment by hizzlekizzle** [comment]
Just that we didn't build them so we can't guarantee they haven't been tampered with, etc. I'm not saying that to cast aspersions, just explaining why we can't ethically distribute them ourselves.

**Comment by theofficialgman** [comment]
@hizzlekizzle can we get any more information on this? I'd like to get these builds updated (and have generic aarch64 builds added officially). Willing to assist where possible.

**Comment by theofficialgman** [comment]
@Libretro bump. As mentioned above, I would like to assist getting aarch64 linux builds added and armv7l linux builds updated

**Comment by Ninoh-FOX** [comment]
I have some cores build for armhf

https://github.com/Rparadise-Team/Koriki/tree/cores/armhf

**Comment by theofficialgman** [comment]
@LibretroAdmin @libretro @hizzlekizzle bump. Happy to help with infrastructure to get these builds update and have aarch64 added!

**Comment by gouchi** [comment]
Related [issue](https://github.com/libretro/libretro-super/issues/1874) on Libretro super repository.
